tests/test_components.py

```diff
@@ -288,7 +288,7 @@ def test_num_mediums():
 
     structures = []
     grid_spec = GridSpec.auto(wavelength=1.0)
-    for i in range(200):
+    for i in range(65530):
```

Comment:
let's import this from `tidy3d.components.simulation`?

Reply:
changed